README.md

```diff
@@ -221,13 +220,33 @@ milestones [here](https://github.com/Candy-Doc/candy-doc-maven-plugin/milestones
 
 The development team works on a branch named _main_.
 
-Whenever a feature needs to be implemented, a new branch must be pulled from
-_main_ and merged to this latter once the feature is ready.
+Whenever a feature needs to be implemented, a new branch (feat/<new_branch>) must be pulled from
+_main_ and rebased to this latter once the feature is ready.
+
+```mermaid
+graph TD
+    .. --main--> 1
+    1 --> 2
+    2 --> 3
+    3 --> 4
+    1 --feat/my-feature--> 2bis
+    2bis --> 3bis
+    3bis --> 4bis
+```
 
-Every time a new version is about to be released, a merge from your new branch to _main_
+Then, every time a new version is about to be released, a rebase from your new branch (feat/<new_branch>) to _main_
```

Comment:
This is not true, rebase doesn't trigger a release or tag creation

Reply:
Done.